resolwe_bio/tests/processes/test_arriba.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import Path
+
+from resolwe.flow.models import Data
+from resolwe.test import tag_process, with_resolwe_host
+
+from resolwe_bio.models import Sample
+from resolwe_bio.utils.test import KBBioProcessTestCase
+
+
+class ArribaProcessorTestCase(KBBioProcessTestCase):
+    @with_resolwe_host
+    @tag_process("arriba")
+    def test_arriba_fusion_detection(self):
+        input_folder = Path("arriba") / "input"
+        output_folder = Path("arriba") / "output"
+
+        with self.preparation_stage():
+            bam = self.run_process(
```

Comment:
@zkuralt @romunov The test needs to be rewritten such that the reads are aligned by STAR. For this, you need to modify the test files such that you extract the reads from the "aligned_samples.bam" and align this data to a suitable STAR index generated by using a matching FASTA file. In this way, also the required `chimeric` output will be created and stored in the STAR-aligned BAM Data object.